.github/workflows/build.yml

```diff
@@ -28,4 +28,24 @@ jobs:
         run: dotnet build --no-incremental /p:SignAssembly=true /p:AssemblyOriginatorKeyFile=../key.snk
 
       - name: Test SDK
-        run: dotnet test
+        run: dotnet test --configuration Release --verbosity normal --collect:"XPlat Code Coverage" --results-directory ./coverage -p:ExcludeByFile="**/GitHub/**/*.cs" 
+
+      - name: Code Coverage Report
+        uses: irongut/CodeCoverageSummary@v1.3.0
```

Comment:
Do you mind explaining why you've gone with this tool? I haven't heard of it before and I'm curious what the alternatives and tradeoffs were.

Reply:
Sure... it was the most streamlined, most starred, I would've liked it to be more recently maintained - some other implementations had gists, repos, and accounts tied to it so while I was evaluating different tooling and trying to decide if we even want to do something like this - it seemed to be the most reasonable option.